src/lib/SapphireClient.ts

```diff
@@ -33,6 +39,7 @@ export class SapphireClient extends Client {
 		super(options);
 
 		this.clientID = options.id ?? null;
+		this.arguments = new ArgumentStore(this).registerPath(join(__dirname, '..', 'arguments'));
 		this.commands = new CommandStore(this);
```

Comment:
Shouldn't this have `.registerPath(join(__dirname, '..', 'commands'))` 🤔 ?

Reply:
no, because there are no built-in commands. Specifying that will cause an error.

Reply:
ah